src/sentry/models/apikey.py

```diff
@@ -120,5 +120,8 @@ def get_scopes(self):
         # All scopes!
         return ApiKey.scopes
 
+    def has_scope(self, scope):
```

Comment:
This looks scary, but it's on the `SystemKey` since it didn't have this method defined.